app/src/main/java/com/example/topmovies/App.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.topmovies
+
+import android.app.Application
+import com.example.topmovies.di.appModule
+import org.koin.android.ext.koin.androidContext
+import org.koin.android.ext.koin.androidLogger
+import org.koin.core.context.startKoin
+import org.koin.core.logger.Level
+
+class App : Application() {
+    override fun onCreate() {
+        super.onCreate()
+        startKoin {
+            androidLogger(Level.DEBUG)
+            androidContext(this@App)
+            modules(appModule)
+        }
+    }
+}
```

Comment:
Please always add a new line at the end.
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
It's problem for C/C++, don't think that we have to do it in Java/Kotlin